src/EditorFeatures/VisualBasic/GoToDefinition/VisualBasicGoToDefinitionSymbolService.vb

```diff
@@ -14,5 +16,31 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.GoToDefinition
         Protected Overrides Function FindRelatedExplicitlyDeclaredSymbol(symbol As ISymbol, compilation As Compilation) As ISymbol
             Return symbol.FindRelatedExplicitlyDeclaredSymbol(compilation)
         End Function
+
+        Protected Overrides Async Function GetDeclarationPositionIfOverride(syntaxTree As SyntaxTree,
+                                                                            position As Integer,
+                                                                            cancellationToken As CancellationToken) As Task(Of Integer?)
+
+            Dim token = Await CodeAnalysis.Shared.Extensions.SyntaxTreeExtensions.GetTouchingTokenAsync(syntaxTree, position, cancellationToken).
+                ConfigureAwait(False)
+
+            If token.Kind() = SyntaxKind.OverridesKeyword Then
+                Dim parent = token.Parent
+
+                Select Case parent.Kind()
+                    Case SyntaxKind.SubStatement, SyntaxKind.FunctionStatement
+                        Dim method = DirectCast(parent, MethodStatementSyntax)
+                        Return method.Identifier.SpanStart
+
+                    Case SyntaxKind.PropertyStatement
+                        Dim [property] = DirectCast(parent, PropertyStatementSyntax)
+                        Return [property].Identifier.SpanStart
```

Comment:
does VB nor have overridden events?

Reply:
Correct. No overridable events in VB

---
In reply to: [152661966](https://github.com/dotnet/roslyn/pull/22963#discussion_r152661966) [](ancestors = 152661966)